### ISSUE TITLE
add possibility to invalidate asset cache for all with the image service created image variants of a base image

### ISSUE THREAD

**Issue by Slackman2015** [storyblok/storyblok#477]
**The feature would affect:** (check one with "x")   
- [x] *app.storyblok.com (CMS - Interface)*   
- [ ] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other* 

**Is your feature request related to a problem? Please describe.**
When using the image service and the replace assets app for images it is necessary to invalidate the CDN cache for the from the base images derived variants manually using the cache invalidation field in the space settings:
https://www.storyblok.com/cl/cache-image-service

**Describe the solution you'd like**
It would be a good addition to be able to invalidate the cache of those with the image service generated images when using the replace assets app.

**Comment by jorgemartins-uon** [comment]
Any updates on this?

**Comment by jarkt** [comment]
I think it's wrong, that the consumer has to care about your CDN. It doesn't care me if you use a CDN. If I upload an image my expectation is just as clear as if I delete an image. If there is a technical need to tell the CDN to delete the image, then handle it inside your application. I don't know why I should care about that.

So I think, it's not a "good addition to be able to invalidate the cache" or an enhancement. It's just a bug as it is.

And the bug could create real problems for your customers. Think of an copyrighted image that you shouldn't have published. Currently you cannot delete it. Not even with a manual request, as it should be "possible" in the space settings. It has no effect. (My image was deleted more than 30 days before and was invalidated in the CDN 8 days before. It still exists.)

BTW: Who pays for unreferenced but still saved images?

**Comment by jorgemartins-uon** [comment]
I agree. Also, while I appreciate the new visual enhancements with Storyblok v2, I feel there are a lot of basic functionality issues opened here that would be way more useful if delivered/fixed. This image cache is a clear example, it should've been tackled a while ago.

**Comment by alvarosabu** [comment]
Hi @Slackman2015, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
